### PR TITLE
Stylized the  MultipleSelectField by design, (hover effect left)

### PR DESCRIPTION
**Pull request by artem98tema**

### CODE REVIEW

**Review by DRITE — CHANGES_REQUESTED** [review]
We use english language for commit messages and descriptions of pull requests.

File: ui/src/fields/MultipleSelectField/MultipleSelectField.tsx [hunk 1]
```diff
@@ -45,8 +46,7 @@ const MultipleSelectField: React.FunctionComponent<MultipleSelectFieldProps> = p
 
     const extendedProps: SelectProps<string[]> = {
         ...props,
-        /* className: styles.container,
```
Comment (DRITE) [inline ui/src/fields/MultipleSelectField/MultipleSelectField.tsx:48]: Ok, you've deleted stale comment. But what about TODO in the file?

**Review by DRITE — CHANGES_REQUESTED** [review]

**Comment by DRITE** [comment]
@artem98tema I repeat: We use english language for commit messages and descriptions of pull requests.

**Comment by DRITE** [comment]
> (hover effect left)

What does it mean? Is it actual?

**Comment by artem98tema** [comment]
> > (hover effect left)
> 
> What does it mean? Is it actual?
there was no effect in the design of the hover, but the designer said to leave

**Comment by DRITE** [comment]
> > > (hover effect left)
> > 
> > 
> > What does it mean? Is it actual?
> > there was no effect in the design of the hover, but the designer said to leave

Could you describe exactly what kind of effect do you mean, please?

**Comment by artem98tema** [comment]
> > > > (hover effect left)
> > > 
> > > 
> > > What does it mean? Is it actual?
> > > there was no effect in the design of the hover, but the designer said to leave
> 
> Could you describe exactly what kind of effect do you mean, please?

this effect 
image

File: ui/src/fields/MultipleSelectField/MultipleSelectField.module.css [hunk 2]
```diff
@@ -1,109 +1,13 @@
...
-/*}*/
+.span{
+  margin-left: 8px;
+}
+:global(.ant-select-dropdown--multiple){
+  max-width: 255px;
+}
+:global(.ant-select-dropdown-menu-item-selected):not(:hover){
+  background-color: white;
+}
```
Comment (DRITE) [inline ui/src/fields/MultipleSelectField/MultipleSelectField.module.css:9]: These selectors aren't scoped under `MultipleSelectField` component and will affect all `<Select>` from ant design components on the page.

**Review by DRITE — CHANGES_REQUESTED** [review]

**Review by DRITE — APPROVED** [review]

**Pull request merged**